Assets/Scripts/Menus/Options/DevPanel.cs

```diff
@@ -96,13 +114,30 @@ public class DevPanel : MonoBehaviour {
     }
 
     public void CopyToClipboard() {
-        GUIUtility.systemCopyBuffer = GetFlightParams().ToJsonString();
+        var json = GetFlightParams().ToJsonString();
+
+        var jObject = JObject.Parse(json);
+        if ((float)jObject.GetValue("mass") == ShipParameters.Defaults.mass)
```

Comment:
if the defaults attribute thing mentioned above was implemented, I _think_ these could be removed (nothing would export unless different to default)